SYS_ATL/LoopIR_unification.py

```diff
@@ -612,7 +612,7 @@ def __init__(self, subproc, stmt_block, live_vars):
 
         self.stride_holes   = { fa.name : False for fa in subproc.args
                                                 if fa.type == T.stride }
-        
+
```

Comment:
If you want small diffs, you'll have to use automatic source formatting (or at _least_ whitespace normalization) tools...